### PR TITLE
Add support for Python 3.13

### PR DESCRIPTION
**Pull request by jesper-friis**
# Description
Add support for Python 3.13.

## Type of change
- [ ] Bug fix and code cleanup
- [x] New feature
- [ ] Documentation update
- [ ] Testing

## Checklist for the reviewer
This checklist should be used as a help for the reviewer.

- [ ] Is the change limited to one issue?
- [ ] Does this PR close the issue?
- [ ] Is the code easy to read and understand?
- [ ] Do all new feature have an accompanying new test?
- [ ] Has the documentation been updated as necessary?
- [ ] Is the code properly tested?

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -136,7 +138,7 @@ good-names = [
 
 [tool.pytest.ini_options]
 minversion = "7.0"
-addopts = "-rs --cov=tripper --cov-report=term --doctest-modules"
+addopts = "-rs --cov=tripper --cov-report=term --doctest-modules --ignore=examples --ignore=tripper"
```
Comment (francescalb) [inline pyproject.toml:141]:
```suggestion
addopts = "-rs --cov=tripper --cov-report=term --doctest-modules --ignore=examples"
```
I also do not like ignoring of examples, but tripper which is our code-base should definitely not be ignored.

Comment (jesper-friis) [reply pyproject.toml]: Replaced it with new `--doctest-ignore-import-errors`

Comment (jesper-friis) [reply pyproject.toml]: > Replaced it with new `--doctest-ignore-import-errors`

Comment (jesper-friis) [reply pyproject.toml]: > I also do not like ignoring of examples, but tripper which is our code-base should definitely not be ignored.

Replaced it with new `--doctest-ignore-import-errors`

**Pull request merged**